Looks like this broke all configurations build.
```
F:\workspace\_work\1\s\.packages\microsoft.dotnet.apicompat\5.0.0-beta.20168.1\build\Microsoft.DotNet.ApiCompat.targets(49,5): error : ResolvedMatchingContract item must be specified to run API compat. [F:\workspace\_work\1\s\src\libraries\Microsoft.Extensions.DependencyInjection\src\Microsoft.Extensions.DependencyInjection.csproj]
##[error].packages\microsoft.dotnet.apicompat\5.0.0-beta.20168.1\build\Microsoft.DotNet.ApiCompat.targets(49,5): error : ResolvedMatchingContract item must be specified to run API compat.
```

This is the first src project that builds that needs to run API compat.  I believe this is because we don't binplace for every configuration.

--- a/eng/resolveContract.targets
+++ b/eng/resolveContract.targets
@@ -10,11 +10,6 @@
 
   <ItemGroup Condition="'$(HasMatchingContract)' == 'true'">
     <ResolvedMatchingContract Condition="Exists('$(ContractAssemblyPath)')" Include="$(ContractAssemblyPath)" />
-    <!-- If the contract doesn't exist in the default contract output path add a project reference to the contract project to resolve -->
-    <ProjectReference Condition="'@(ResolvedMatchingContract)' == ''" Include="$(ContractProject)">
-      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-      <OutputItemType>ResolvedMatchingContract</OutputItemType>
-    </ProjectReference>
   </ItemGroup>
 
   <!-- intentionally empty since we no longer need a target -->

